### PR TITLE
Update noscript.md

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Removed an irrelevant part

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
Looks good to go.

File: content/html/concepts/tags/terms/noscript/noscript.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-The `<noscript>` tag displays its contents whenever [Javascript](https://www.codecademy.com/resources/docs/javascript) is disabled on the browser or it is not supported for a given site.
+No The `<noscript>` tag displays its contents whenever [Javascript](https://www.codecademy.com/resources/docs/javascript) is disabled on the browser.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/noscript/noscript.md:14]: @ethan-master-coding ??

**Review by Dusch4593 — COMMENTED** [review]
@ethan-master-coding Just had one comment; wondering if that change was meant to go on another line?

**Comment by ethan-master-coding** [comment]
@Dusch4593 That's what happens when you do stuff on mobile

**Review by Dusch4593 — APPROVED** [review]

**Pull request merged**